interfaces/builtin/pluggable_storage.go

```diff
@@ -0,0 +1,40 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const pluggableStorageConnectedPlugAppArmor = `
+# Description: Can access removable storage filesystems
+
+# Filesystems are mounted to /run/media/<user>/foo since /media is read-only
+/run/media/** rw,
+`
+
+// NewPluggableStorageInterface returns a new "pluggable-storage" interface.
+func NewPluggableStorageInterface() interfaces.Interface {
+	return &commonInterface{
+		name: "pluggable-storage",
+		connectedPlugAppArmor: pluggableStorageConnectedPlugAppArmor,
+		reservedForOS:         true,
```

Comment:
My intention here is that a snap with this plug connected can use the mounted filesystems (which I can move to /media per your above comment) My thought was that a snap wanting to use pluggable storage wouldn't actually have to give a rip about how that filesystem was mounted (whether via udisks or manual or whatever) just that it would be able to see that something was there and make use of it.